concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/contractexample/parameters/Transfer.java

```diff
@@ -0,0 +1,28 @@
+package com.concordium.sdk.examples.contractexample.parameters;
+
+import com.concordium.sdk.types.AbstractAddress;
+import com.concordium.sdk.types.UInt8;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+
+/**
+ * Represents a '<a href = "https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/concordium-cis2/src/lib.rs#L1073">Transfer</a>' used in different smart contracts
+ */
+@Getter
+@AllArgsConstructor
+public class Transfer<T extends TokenId, A extends TokenAmount> {
```

Comment:
I used the name `Transfer` to match the struct in https://github.com/Concordium/concordium-rust-smart-contracts/blob/bc7e7ae7db029e78c0f1e965c968ca1364f4ce1c/concordium-cis2/src/lib.rs#L1081C18-L1081C18

Reply:
Yes, but that origins in the CIS2 library. If we at some point want to add another example which has some other `Transfer` then it can end up being a bit confusing. Better to just be explicit in these examples.

Reply:
I see, that makes total sense!